lua/possession/session.lua

```diff
@@ -229,6 +229,10 @@ function M.load(name_or_data, opts)
     local path
     if type(name_or_data) == 'string' then
         path = paths.session(name_or_data)
+        if not path:exists() then
```

Comment:
Should we prefer `vim.fn.filereadable` here instead of `path:exists()`?

I wasn't able to find the correct path name from the path table to make `filereadable` work, but if you know how, happy to adjust this line.

Reply:
Well, we could, but I think it's actually better to leave `:exists()` here. Consider the options:
* the file does not exist and we use `:exists()` - user gets a useful error message
* the file does not exist and we use `filereadable()` - user gets message 'file not readable' which is fine 
* file exists but is not readable or is a directory and we use `:exists()` - user gets error `EACCES: permission denied` or `ENOENT: no such file or directory` - a little ugly because with traceback but gives correct reason
* file exists but is not readable and we use `filereadable()` - user gets message 'file not readable' which is good
* file exists but is a directory and we use `filereadable()` - user gets message 'file not readable' which is misleading